src/test/java/tests/BaseTest.java

```diff
@@ -0,0 +1,31 @@
+package tests;
+
+import bo.CartActions;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import factory.DriverFactory;
+import modules.CartPageModule;
+import org.testng.annotations.AfterClass;
+
+import org.testng.annotations.BeforeClass;
+
+import page.CartWebPage;
+import validators.CartValidator;
+
+public class BaseTest {
+    protected CartActions cartActions;
```

Comment:
why do you need this two classes in BaseTest

Reply:
this classes I should create in CartTest?

Reply:
but not in the BaseTest. You should use @Inject for Actions and Validators as well as for Pages.
Don't use 'new'

Reply:
okey, where I should inject Page , in Actions, 
and Actions ,Validators I inject in Test  ?

Reply:
fixed